IssueTracker/IssueTracker.FileSystem/Repository/IRepository/IBolbData.cs

```diff
@@ -1,7 +1,10 @@
-﻿namespace IssueTracker.FileSystem;
+﻿using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("IssueTracker.UnitTest")]
+namespace IssueTracker.FileSystem;
 internal interface IBolbData
 {
     Task<IEnumerable<Models.File>> GetFilesAsync(IEnumerable<Models.File> files);
     Task UploadFileAsync(Models.File file);
-    Task<Stream> DownloadFileAsync(string link);
+    Task<bool> DeleteFileAsync(string name);
```

Comment:
The name can be simplified. DeleteAsync()

Reply:
done